hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/scm/TestStorageContainerManager.java

```diff
@@ -986,6 +989,7 @@ public void testContainerReportQueueWithDrop() throws Exception {
   }
 
   @Test
+  @Category(FlakyTest.class) @Flaky("HDDS-8470")
```

Comment:
Only Category tag is sufficient for the flaky job to pick it up right?

Reply:
Did we add flaky tag only to label the jira?

Reply:
Correct.  `@Category(FlakyTest.class)` causes the test to be moved to _integration (flaky)_, and `@Flaky(...)` is added only to reference the Jira issue.